### PR TITLE
Added Address Submission to Readme file

### PR DESCRIPTION
**Pull request by PieterCachet**
Added Address Submission to Readme file

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -273,3 +273,29 @@ final Response response = serviceFactory.get(request);
...
+	<artifactId>addresssubmission</artifactId>
+	<version>3.0.1</version>
+</dependency>
+```
+
+
+```java
+import com.afrigis.services.AfriGISServices;
+import com.afrigis.services.ServiceCallFactory;
+import com.afrigis.services.geocode.AddressRequest;
```
Comment (demaniak) [inline readme.md]: These imports can't be right?

File: readme.md [hunk 2]
```diff
@@ -273,3 +273,29 @@ final Response response = serviceFactory.get(request);
...
+import com.afrigis.services.geocode.AddressResponse;
+
+ServiceCallFactory serviceFactory = AfriGISServices.instance("Key","Secret");
+AddressSubmissionRequest addressSubParams = new AddressSubmissionRequest("KEY", "Address", "AdditionalInfomation");
```
Comment (demaniak) [inline readme.md]: What is the `KEY` in this context?

Maybe just add a comment to make it clear it is your SaaS key?

File: readme.md [hunk 3]
```diff
@@ -273,3 +273,29 @@ final Response response = serviceFactory.get(request);
...
+AddressSubmissionResponse response = serviceFactory.get(addressSubParams);
+List<SubmissionResult> addressSubmList = response.listResults();
+System.out.println (addressSubmList.get(0).getClientID());
```
Comment (demaniak) [inline readme.md]: Please explicitly extract the SubmissionResult instance and demonstrate using it:
```
SubmissionResult result = addressSubmList.get(0);
System.out.println (result.getClientID());

**Review by demaniak — CHANGES_REQUESTED** [review]
Almost there, please just make indicated changes, thanks.

File: readme.md (hunk 3 above)
Comment (PieterCachet) [reply readme.md]: Done

**Comment by PieterCachet** [comment]
@demaniak done

**Review by demaniak — APPROVED** [review]
LGTM

**Pull request merged**